doc/whats-new.rst

```diff
@@ -85,6 +85,10 @@ Breaking changes
   as positional, all others need to be passed are keyword arguments. This is part of the
   refactor to support external backends (:issue:`4309`, :pull:`4989`).
   By `Alessandro Amici <https://github.com/alexamici>`_.
+- :py:func:`mean` does not change the data if axis is None. This
```

Comment:
:py:func:`mean` is actually a method (there should be other references in the whats-new which can be copied)

Reply:
I haven't found an example, so I have removed the explicit reference